test/unit/spec/ConsentRequired_spec.js

```diff
@@ -50,6 +54,16 @@ function (Q, _, $, OktaAuth, LoginUtil, SharedUtil, Util, ConsentRequiredForm, E
     return Expect.waitForConsentRequired(settings);
   }
 
+  function setupClientLogo() {
+    var resConsentRequiredClientLogo = deepClone(resConsentRequired);
+    var customLogo = {
+      href: 'https://example.com/logo.png',
```

Comment:
nit nit: ```.com/custom-logo.png``` so when we see the test in line 82 is really obvious that we are expecting a custom logo